src/Vegetation/photosynthesis.jl

```diff
@@ -192,3 +221,54 @@ function compute_photosynthesis(
     Rd = dark_respiration(Vcmax25, β, f, ΔHRd, T, To, R)
     return net_photosynthesis(Ac, Aj, Rd, β)
 end
+
+function compute_Jmax25(Jmax_model::PrescribedJmax25, _...)
+    return Jmax_model.Jmax25
+end
+
+
+function compute_Jmax25(Jmax_model::OptimalityJmax25, parameters::FarquharParameters, T, medlyn_factor,
+                         APAR, c_co2, R)
+    """Solve for Jmax25 assuming Aj=Ac"""
+    (;
+        Γstar25,
+        ΔHJmax,
+        ΔHVcmax,
+        ΔHΓstar,
+        f,
+        ΔHRd,
+        To,
+        θj,
+        ϕ,
+        mechanism,
+        sc,
+        ψc,
+        oi,
+        Kc25,
+        Ko25,
+        ΔHkc,
+        ΔHko,
+        jv_ratio,
+    ) = parameters
+
+    Γstar = co2_compensation(Γstar25, ΔHΓstar, T, To, R)
+    ci = intercellular_co2(c_co2, Γstar, medlyn_factor)
+    Kc = MM_Kc(Kc25, ΔHkc, T, To, R)
+    Ko = MM_Ko(Ko25, ΔHko, T, To, R)
+
+    # Light utilization of APAR
+    IPSII = ϕ * APAR / 2
+
+    Jmax_arr = arrhenius_function(T, To, R, ΔHJmax)
+    Vcmax_arr = arrhenius_function(T, To, R, ΔHVcmax)
+
+    if typeof(mechanism) == C3
```

Comment:
Im not sure if this would be confusing, but these lines can be reduced to:
```
FT = typeof(jv_ratio)
C = rubisco_assimilation(mechanism, FT(1), ci, Γstar, Kc, Ko, oi) * Vcmax_arr/Jmax_arr/jv_ratio
```

Reply:
Good point! But yes, I think it may be confusing.